samples/beam/requirements.txt

```diff
@@ -1,3 +1,3 @@
 apache-beam==2.28.0
-google-cloud-bigtable<2.0.0dev1
+google-cloud-bigtable<2.0.0
```

Comment:
I think this will include the dev candidate in the allowed versions. This might be unintended?

Reply:
I think it's ok - if it were picking up the dev candidate the build would fail, like in #283